pandas/plotting/_tools.py

```diff
@@ -311,7 +311,7 @@ def _handle_shared_axes(axarr, nplots, naxes, nrows, ncols, sharex, sharey):
                     # only the last row of subplots should get x labels -> all
                     # other off layout handles the case that the subplot is
                     # the last in the column, because below is no subplot/gap.
-                    if not layout[ax.rowNum + 1, ax.colNum]:
+                    if not layout[ax.rowNum + 1, ax.colNum] or getattr(ax, '_pandas_colorbar_axes', False):
```

Comment:
This line is too long: https://travis-ci.org/pandas-dev/pandas/jobs/359516939#L3006

Reply:
Thanks. I'll fix it. Sorry I'm pretty new to the open-source world.